lib/vanagon/component.rb

```diff
@@ -111,7 +112,8 @@ def get_source(workdir) # rubocop:disable Metrics/AbcSize
         @source = Vanagon::Component::Source.source(url, opts)
         source.fetch
         source.verify
-        @extract_with = source.respond_to?(:extract) ? source.extract(platform.tar) : ':'
+        temp = source.respond_to?(:extract) ? source.extract(platform.tar) : ':'
+        @extract_with = [temp]
```

Comment:
I think we can get rid of this change and just leave it 
`@extract_with = source.respond_to?(:extract) ? source.extract(platform.tar) : ':'` since we're going to make it an array in get_sources anyways